resources/js/Components/Selects/Interface.tsx

```diff
@@ -0,0 +1,37 @@
+
+
+export interface IInputTextProps {
```

Comment:
I think it was cool to create a separate interface folder and put the interfaces data there
@am0nshi

Reply:
remove  to folder Interfaces